sdk/_version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.0.8"
+__version__ = "0.0.9"
```

Comment:
Nit out of curiosity, which kind of changes trigger a version update ?

Reply:
Currently a github action is set up to release a new version of the package every time the dev branch is updated (i.e. after each merge request.

So anytime we want to release a new version of the SDK we manually bump the version here, and it will automatically get built and released when merged. 

When the SDK is more stable we will release a new package version more sporadically. We will probably move the github action to the master branch, and only bump version when merging the dev branch onto master.